extra-multimedia/feeluown-kuwo/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME="feeluown-kuwo"
+PKGSEC=sound
+PKGDES="a FeelUOwn plugin for KuWo"
```

Comment:
First letter in `PKGDES=` should be capitalised; the "A" is actually not necessary here.